examples/static_react_task_with_tips/webapp/src/components/core_components.jsx

```diff
@@ -79,6 +79,24 @@ function SimpleFrontend({ taskData, isOnboarding, onSubmit, onError }) {
         <div style={{ margin: "15rem 2rem 2rem auto", width: "fit-content" }}>
           <Tips width="30rem" maxHeight="30rem" />
         </div>
+        <div style={{ margin: "10rem 0 0 0", width: "min(25rem, 100%)" }}>
+          <Tips
+            width="25rem"
+            maxHeight="30rem"
+            placement="top-start"
+            maxHeaderLength={30}
+            maxTextLength={300}
+          />
+          <div style={{ margin: "1rem 0 0" }}>
+            <Feedback
+              maxTextLength={30}
+              questions={[
+                "What is your favorite part of this task?",
+                "Were you satisfied with this task?",
+              ]}
+            />
```

Comment:
The new Feedback component, you can optionally \ass in questions or have no questions and just have general feedback.